postgresql-11.5/src/bin/psql/commandbteq.c

```diff
@@ -796,4 +781,28 @@ process_file_bteq(char *filename, bool use_relative_path)
 
     pset.inputfile = oldfilename;
     return result;
-}
\ No newline at end of file
+}
+
+
+/*
+ * Ask the user for a password; 'username' is the username the
+ * password is for, if one has been explicitly specified. Returns a
+ * malloc'd string.
+ */
+static char *
+prompt_for_password(const char *username)
```

Comment:
We don't need it here. It should be added with LOGON PR.

Reply:
we didn't need it,i deleted it